zh/coding-standard.md

```diff
@@ -563,18 +559,18 @@ String.prototype.trim = function () {
 
 ### 变量
 
-**[强制]** 变量必须用 `var` 声明，且一个 `var` 不能同时声明多个变量。
+**[强制]** 优先使用`const`声明变量 且一行不能同时声明多个变量。
```

Comment:
Should add one `，`. The others look good to me.

Reply:
@plainheart Thanks, comma has added